src/Controller/SantaController.php

```diff
@@ -127,13 +127,28 @@ public function message(Rudolph $rudolph, Request $request, string $application)
         $selectedUsers = $session->get('selected-users', []);
         $message = $session->get('message');
         $notes = $session->get('notes');
+        $options = $session->get('options');
 
         $errors = [];
 
         if ($request->isMethod('POST')) {
             $message = trim($request->request->get('message'));
             $notes = array_map('trim', $request->request->all('notes'));
 
+            $options = [];
+            $scheduled = false;
+
+            if ('' !== $request->request->get('scheduled_at')) {
```

Comment:
`$request->request->get('scheduled_at')` return `null` by default, so testing for "not empty" is not enough.

You could check this like that:

```suggestion
            if ($request->request->get('scheduled_at')) {
```

That way, if the field does not exists, or if it's empty, it will mean `false`.

Reply:
Thanks for the suggested code, I tested it and committed your change 😄